sources/packages/api/src/database/entities/sin-validation.model.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  RelationId,
+} from "typeorm";
+import { User } from ".";
+import { ColumnNames, TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+
+/**
+ * SIN Validations that must be performed with CRA.
+ */
+@Entity({ name: TableNames.SINValidation })
+export class SINValidation extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+  /**
+   * Date and time that the SIN validation file was generated
+   * and sent.
+   */
+  @Column({
+    name: "date_sent",
+    nullable: true,
```

Comment:
please add the type too
![image](https://user-images.githubusercontent.com/77353155/157527151-34777cb5-bd0f-42cf-a906-bfadfb299377.png)